apps/frontend/src/components/fap/MeetingComponents/FapMeetingInstrumentsTable.tsx

```diff
@@ -163,6 +200,32 @@ const FapMeetingInstrumentsTable = ({
             }
           )(),
         tooltip: 'Submit ' + i18n.format(t('instrument'), 'lowercase'),
+      }),
+      (
+        rowData:
+          | InstrumentWithAvailabilityTime
+          | InstrumentWithAvailabilityTime[]
+      ) => ({
+        icon: RemoveDoneIcon,
+        hidden: !(rowData as InstrumentWithAvailabilityTime).submitted,
+        onClick: (
+          event: Event,
+          rowData:
+            | InstrumentWithAvailabilityTime
+            | InstrumentWithAvailabilityTime[]
+        ) =>
+          confirm(
+            () => {
+              unsubmitInstrument(rowData as InstrumentWithAvailabilityTime);
+            },
+            {
+              title: 'Unsubmit ' + i18n.format(t('instrument'), 'lowercase'),
+              description: `This action will reopen the proposal reordering in the meeting components. Are you sure you want to unsubmit the ${t(
+                'instrument'
+              )}?`,
```

Comment:
Should this end with `} instrument?` or `} experiments?` instead of `}?`.

I think `Are you sure you want to unsubmit the LOKI instrument?` is better gramatically than `Are you sure you want to unsubmit the LOKI?`

And it may just be lack of familiarity with the framework, but it looks like this message title & tooltip have a lowercase instrument name while the description contains the original. I think it'd be better if they were consistent.

Reply:
I agree

Reply:
The `i18n.format(t('instrument')` does not select the instrument name but it is just used for internationalization. So the word `instrument` might be shown differently for different configurations and we want it to be lowercase. But I will add the name of the instrument/experiment there now to be more clear what are you trying to submit and unsubmit.